editoast/schemas/src/fixtures.rs

```diff
@@ -35,38 +34,38 @@ use crate::train_schedule::Margins;
 use crate::train_schedule::TrainScheduleOptions;
 
 pub fn simple_rolling_stock() -> RollingStock {
-    RollingStock {
-        name: "SIMPLE_ROLLING_STOCK".to_string(),
-        loading_gauge: LoadingGaugeType::G1,
-        supported_signaling_systems: RollingStockSupportedSignalingSystems(vec![]),
-        base_power_class: None,
-        comfort_acceleration: units::meter_per_second_squared::new(0.1),
-        inertia_coefficient: 1.10,
-        startup_acceleration: units::meter_per_second_squared::new(0.04),
-        startup_time: units::second::new(1.0),
-        effort_curves: EffortCurves::default(),
-        electrical_power_startup_time: None,
-        raise_pantograph_time: None,
-        energy_sources: vec![],
-        const_gamma: units::meter_per_second_squared::new(1.0),
-        etcs_brake_params: None,
-        metadata: None,
-        power_restrictions: HashMap::new(),
-        railjson_version: "12".to_string(),
-        rolling_resistance: RollingResistance {
+    RollingStock::new(
+        "SIMPLE_ROLLING_STOCK".to_string(),
+        EffortCurves::default(),
+        None,
+        units::meter::new(140.0),
+        units::meter_per_second::new(20.0),
+        units::second::new(1.0),
+        units::meter_per_second_squared::new(0.04),
+        units::meter_per_second_squared::new(0.1),
+        units::meter_per_second_squared::new(1.0),
+        None,
+        1.1,
+        units::kilogram::new(15000.0),
+        RollingResistance {
             rolling_resistance_type: "davis".to_string(),
-            // TODO those values are wrong, they correspond to daN/T, (daN/T)/(km/h), and (daN/T)/(km/h)²
+            // TODO those values are wrong, they correspond to daN/T, (daN/T)/(km/h), and (daN/T)/(km/h)² per weight
```

Comment:
looks like a bad paste:
```suggestion
            // TODO those values are wrong, they correspond to daN/T, (daN/T)/(km/h), and (daN/T)/(km/h)²
```